### PR TITLE
uudecode: add page

### PR DESCRIPTION
**Pull request by bl-ue**
Apologies for the _long_ delay @marchersimon... 😢

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — COMMENTED** [review]
What about `--output-file`, `--help` and `--mode-help` arguments?

File: pages/common/uudecode.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# uudecode
+
+> Decode files encoded by `uuencode`.
+> More information: <https://manned.org/uudecode>.
+
+- Decode a file that was create with `uuencode` into the current directory:
+
+`uudecode {{path/to/file}}`
```
Comment (marchersimon) [inline pages/common/uudecode.md:8]:
```suggestion
- Decode a file that was encoded with `uuencode` and print the result to stdout:

`uudecode {{path/to/encoded_file}}`

- Decode a file that was encoded with `uuencode` and write the result to a file:

`uudecode -o {{path/to/decoded_file}} {{path/to/encoded_file}}`
```
Maybe we should keep it consistent with the `uuencode` page.

Comment (marchersimon) [reply pages/common/uudecode.md:8]: Also, the file name of the encoded file has to be the one, used when encoding the file. How should we mention that?

Comment (CleanMachine1) [reply pages/common/uudecode.md:8]: Why are there duplicate commands

File: pages/common/uudecode.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# uudecode
+
+> Decode files encoded by `uuencode`.
+> More information: <https://manned.org/uudecode>.
+
+- Decode a file that was create with `uuencode` into the current directory:
```
Comment (nicokosi) [inline pages/common/uudecode.md:6]:
```suggestion
- Decode a file that was created with `uuencode` into the current directory:
```

Comment (nicokosi) [reply pages/common/uudecode.md:6]: (oops, I prefer @marchersimon's proposal in [the comment below](https://github.com/tldr-pages/tldr/pull/5971/files#r633004204)) 😊

**Comment by bl-ue** [comment]
>What about `--output-file`, `--help` and `--mode-help` arguments?

@navarroaxel the `uudecode` bundled with macOS is _ritarded_.  It dates from January 2002 and has no long options :/

I'll move these to `osx/` later when I write `linux/` pages for them 😉

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Looks good, thanks

**Pull request merged**